### ISSUE TITLE
3rd Party Cookies causing issues?

### ISSUE THREAD

**Issue by Brentless** [microsoftgraph/msgraph-training-spfx#3]
So, maybe 1/4 of our users started having a popup that says "Additional credentials are required to show this page correctly" when we tried to roll out just the Profile web part.

After some research, it seems that this is specifically tied to the 3rd party cookies settings of a user.  If it is set to Allow, everything seems to work great.  But it if is set to prompt or block.  There are all kinds of crazy results.  Some users can just click ok at the popup and everything loads.  Some people end up going into an infinite loop of refreshing.

Are 3rd party cookies the only way this works?  Or is there something that can be done to get around that setting on users browsers?

**Comment by andrewconnell** [comment]
Nope... not to the best of my knowledge. This is how Azure AD is handling auth. The popups are the AAD postback handler which is the target from AAD with the token. The wrapped Graph client that SPFx is exposing leverages the AAD client to acquire the OAuth token needed for calling the MSFT Graph.

In the future suggest posting questions on SPFx => the github.com/sharepoint/sp-dev-docs repo.
